terraform/stacks/bastion_ecr/main.tf

```diff
@@ -102,6 +102,24 @@ resource "aws_ecr_repository_policy" "star_align_nf_cross_accounts" {
   policy     = templatefile("policies/cross_accounts_policy_umccr.json", {})
 }
 
+
+####
+# sash
+#
+
+resource "aws_ecr_repository" "sash" {
+  name                 = "sash"
+  image_tag_mutability = "MUTABLE"
+  image_scanning_configuration {
+    scan_on_push = false
+  }
+}
+
+resource "aws_ecr_repository_policy" "sash_cross_accounts" {
+  repository = aws_ecr_repository.sash.name
+  policy     = templatefile("policies/cross_accounts_policy_umccr.json", {})
+}
+
```

Comment:
ah, looks like `star_align_nf_lifecycle` policy block got pushed down (i.e. as sash blocks get inserted)

Could you double check that please..?

Also, you would want `sash_lifecycle` policy for untagged images like others..?

Reply:
Thanks for catching that, fixed!